src/unet_tf_v1

```diff
@@ -0,0 +1,95 @@
+        tf.compat.v1.layers.conv2d(inputs=conv4, filters=256, kernel_size=[5, 5],
+                                   strides=[2, 2], padding="same", activation=tf.nn.leaky_relu))
+
+    # Convolutional Layer 6
+    conv6 = tf.compat.v1.layers.BatchNormalization()(
+        tf.compat.v1.layers.conv2d(inputs=conv5, filters=512, kernel_size=[5, 5],
+                                   strides=[2, 2], padding="same", activation=tf.nn.leaky_relu))
+
+    # Deconvolutional Layer1 (dropout)
+    deconv1 = tf.compat.v1.layers.BatchNormalization()(
+        tf.compat.v1.layers.conv2d_transpose(inputs=conv6, filters=256, kernel_size=[5, 5],
+                                             strides=[2, 2], padding="same", activation=tf.nn.relu))
+    dropout1 = tf.compat.v1.layers.dropout(inputs=deconv1, rate=0.5, training=mode == tf.estimator.ModeKeys.TRAIN)
+
+    # Deconvolutional Layer2 (dropout)
+    deconv2 = tf.compat.v1.layers.BatchNormalization()(
+        tf.compat.v1.layers.conv2d_transpose(inputs=tf.compat.v1.concat([dropout1, conv5], 3), filters=128,
+                                             kernel_size=[5, 5],
+                                             strides=[2, 2], padding="same", activation=tf.nn.relu))
+    dropout2 = tf.compat.v1.layers.dropout(inputs=deconv2, rate=0.5, training=mode == tf.estimator.ModeKeys.TRAIN)
+
+    # Deconvolutional Layer3 (dropout)
+    deconv3 = tf.compat.v1.layers.BatchNormalization()(
+        tf.compat.v1.layers.conv2d_transpose(inputs=tf.compat.v1.concat([dropout2, conv4], 3), filters=64,
+                                             kernel_size=[5, 5],
+                                             strides=[2, 2], padding="same", activation=tf.nn.relu))
+    dropout3 = tf.compat.v1.layers.dropout(inputs=deconv3, rate=0.5, training=mode == tf.estimator.ModeKeys.TRAIN)
+
+    # Deconvolutional Layer4
+    deconv4 = tf.compat.v1.layers.BatchNormalization()(
+        tf.compat.v1.layers.conv2d_transpose(inputs=tf.compat.v1.concat([dropout3, conv3], 3), filters=32,
+                                             kernel_size=[5, 5],
+                                             strides=[2, 2], padding="same", activation=tf.nn.relu))
+
+    # Deconvolutional Layer5
+    deconv5 = tf.compat.v1.layers.BatchNormalization()(
+        tf.compat.v1.layers.conv2d_transpose(inputs=tf.compat.v1.concat([deconv4, conv2], 3), filters=16,
+                                             kernel_size=[5, 5],
+                                             strides=[2, 2], padding="same", activation=tf.nn.relu))
+    # Deconvolutional Layer6
+    deconv6 = tf.compat.v1.layers.conv2d_transpose(inputs=tf.compat.v1.concat([deconv5, conv1], 3), filters=1,
+                                                   kernel_size=[5, 5],
+                                                   strides=[2, 2], padding="same", activation=tf.nn.relu)
+
+    predictions = {'outputs': deconv6
+                   }
+
```

Comment:
The final layer in the paper uses  __sigmoid__ activation function. Does that mean deconv6 is the mask layer, which is also the final layer?